source/sass/blocks/page-body.scss

```
@@ -1,4 +1,11 @@
 .page-body {
-  background-color: #FFFFFF;
-  font-family: "Open Sans", Arial, sans-serif;
+  width: 320px;
```

Не стоит из элемента `body` делать обёртку. К тому же дизайн макета таков, что общий блок-обёртку применить нельзя, так как есть области, заливка которых тянется на всю ширину экрана и мы не должны её ограничивать.

```suggestion
  min-width: 320px;
```
Это нужно для того, чтобы ширина `body` была не меньше данного значения.